### PR TITLE
chore(repo): fix npm@7 publish for e2e

### PR DESCRIPTION
**Pull request by bekos**

### CODE REVIEW

File: packages/react/package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "eslint-plugin-import": "^2.20.1",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.18.3",
-    "eslint-plugin-react-hooks": "^2.4.0",
+    "eslint-plugin-react-hooks": "4.2.0",
```
Comment (FrozenPandaz) [inline packages/react/package.json:43]: Just wondering.. why did this change? It seems unrelated :thinking:

File: scripts/e2e.ts [hunk 2]
```diff
@@ -29,6 +29,21 @@ function publishPackage(packagePath) {
   }
   try {
     console.log(` 📦 ${packagePath}`);
+
+    // NPM@7 requires a token to publish, thus, is just a matter of fake a token to bypass npm.
+    // See: https://twitter.com/verdaccio_npm/status/1357798427283910660
+    if (npmMajorVersion === 7) {
```
Comment (FrozenPandaz) [inline scripts/e2e.ts:35]: Does this break for `npm@6`? Also, if we need to check then we should do greater or equals so that it doesn't break later on.

**Review by FrozenPandaz — CHANGES_REQUESTED** [review]
Fixes the issue :smile: I had a few questions though.

File: packages/react/package.json (hunk 1 above)
Comment (bekos) [reply packages/react/package.json:43]: The old version has an unmeet peer dependency for eslint, which makes the e2e with npm 7 to fail. Seems like we have already upgraded to the latest version.

File: scripts/e2e.ts (hunk 2 above)
Comment (bekos) [reply scripts/e2e.ts:35]: I think it doesn't break for `npm@6`. I just wanted to be explicit for this, as a workaround :) and I'm pretty confident that it another workaround will be needed when `npm@8` is released :D
